onyo/commands/set.py

```diff
@@ -60,13 +60,13 @@
 
 def set(args: argparse.Namespace) -> None:
     """
-    Set ``KEY``\s to ``VALUE`` for assets.
+    Set **KEY**\s to **VALUE**\s for assets.
 
-    KEY names can be any valid YAML key-name. If a KEY does not exist, it is
-    added and set appropriately.
+    **KEY** names can be any valid YAML key-name. If a key is not present in an
+    asset, it is added and set appropriately.
 
-    Setting KEYs used in the asset name require the ``--rename`` flag, and will
-    result in the asset being renamed on the filesystem.
+    Setting **KEY**\s that are used in the asset name requires the ``--rename``
+    flag, and will result in the asset being renamed on the filesystem.
```

Comment:
Nitpick: maybe instead of
"... and will result in the asset being renamed on the filesystem." 
->
"... and will result also in the asset file or directory being renamed."
Or something similar. It is a bit more clunky naming both types of assets, but I find the usage of "filesystem" a bit overly precise, because we never use that word otherwise and it sounds to me like it happens "just" on the file-system (i.e. as if we opose it to something different that would normaly happen) and not in git or in the asset contents?
I am struggling to formulate my concern, but the word just does not feel that good to me, since we never highlight the "filesystem" part in other scenarios.

Feel free to ignore this comment if it sounds to you like I am just too tired, please!

Reply:
I see your point. The language could certainly be tighter. I will simplify this to:

> Setting **KEY**\s that are used in the asset name requires the ``--rename`` flag.

Reply:
Fixed and pushed.